libfreerdp/crypto/test/TestFuzzCryptoGetDNSNames.c

```diff
@@ -0,0 +1,9 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	X509* certificate = (X509*)Data;
+	int count = 5;
+	crypto_cert_get_dns_names(certificate, &count, 0);
```

Comment:
`crypto_cert_get_dns_names` requires correct (aka non `NULL`) arguments. don´t forget to free return values.